pluginframework/tfschema/attribute_builder.go

```diff
@@ -0,0 +1,42 @@
+package tfschema
```

Comment:
Let's move all of this to `internal/pluginframework/tfschema`.

Reply:
sounds good, moved the entire `pluginframework` folder under `internal`